### PR TITLE
rails 5 production environment fix

### LINKED ISSUE #56: Error in rails 5.2
Hi guys,

I'm trying to use this gem in rails 5.2.  When I do it in development environment it works fine, but when I do it in production I have the following error. Do you have any idea?

Thanks a lot!

logs:
```
RAILS_ENV=production rails s
=> Booting Puma
=> Rails 5.2.0 application starting in production 
=> Run `rails server -h` for more startup options
Exiting
/home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/callbacks.rb:355:in `check_conditionals': Passing string to be evaluated in :if and :unless conditional options is not supported. Pass a symbol for an instance method, or a lambda, proc or block, instead. (ArgumentError)
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/callbacks.rb:301:in `initialize'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/callbacks.rb:289:in `new'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/callbacks.rb:289:in `build'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/callbacks.rb:672:in `block in set_callback'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/callbacks.rb:671:in `map'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/callbacks.rb:671:in `set_callback'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activemodel-5.2.0/lib/active_model/callbacks.rb:131:in `block in _define_before_model_callback'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/invoicing-1.0.1/lib/invoicing/currency_value.rb:122:in `block in <module:CurrencyValue>'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/concern.rb:122:in `class_eval'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/concern.rb:122:in `append_features'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/invoicing-1.0.1/lib/invoicing/class_info.rb:75:in `include'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/invoicing-1.0.1/lib/invoicing/class_info.rb:75:in `acts_as'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/invoicing-1.0.1/lib/invoicing/currency_value.rb:116:in `acts_as_currency_value'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/invoicing-1.0.1/lib/invoicing/ledger_item.rb:315:in `acts_as_ledger_item'
	from /home/aguirrel/proyectos/dedipago-rails/app/models/invoicing_ledger_item.rb:2:in `<class:InvoicingLedgerItem>'
	from /home/aguirrel/proyectos/dedipago-rails/app/models/invoicing_ledger_item.rb:1:in `<main>'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/loaded_features_index.rb:65:in `register'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:283:in `block in require'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:249:in `load_dependency'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:283:in `require'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:370:in `block in require_or_load'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:37:in `block in load_interlock'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies/interlock.rb:14:in `block in loading'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/concurrency/share_lock.rb:151:in `exclusive'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies/interlock.rb:13:in `loading'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:37:in `load_interlock'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:348:in `require_or_load'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:326:in `depend_on'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/active_support.rb:59:in `depend_on'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:242:in `require_dependency'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/engine.rb:478:in `block (2 levels) in eager_load!'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/engine.rb:477:in `each'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/engine.rb:477:in `block in eager_load!'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/engine.rb:475:in `each'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/engine.rb:475:in `eager_load!'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/engine.rb:356:in `eager_load!'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/application/finisher.rb:69:in `each'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/application/finisher.rb:69:in `block in <module:Finisher>'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/initializable.rb:32:in `instance_exec'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/initializable.rb:32:in `run'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/initializable.rb:61:in `block in run_initializers'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/2.4.0/tsort.rb:228:in `block in tsort_each'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/2.4.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/2.4.0/tsort.rb:431:in `each_strongly_connected_component_from'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/2.4.0/tsort.rb:349:in `block in each_strongly_connected_component'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/2.4.0/tsort.rb:347:in `each'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/2.4.0/tsort.rb:347:in `call'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/2.4.0/tsort.rb:347:in `each_strongly_connected_component'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/2.4.0/tsort.rb:226:in `tsort_each'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/2.4.0/tsort.rb:205:in `tsort_each'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/initializable.rb:60:in `run_initializers'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/application.rb:361:in `initialize!'
	from /home/aguirrel/proyectos/dedipago-rails/config/environment.rb:5:in `<main>'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/loaded_features_index.rb:65:in `register'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:283:in `block in require'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:249:in `load_dependency'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:283:in `require'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:44:in `require_relative'
	from config.ru:3:in `block in <main>'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/rack-2.0.5/lib/rack/builder.rb:55:in `instance_eval'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/rack-2.0.5/lib/rack/builder.rb:55:in `initialize'
	from config.ru:in `new'
	from config.ru:in `<main>'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/rack-2.0.5/lib/rack/builder.rb:49:in `eval'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/rack-2.0.5/lib/rack/builder.rb:49:in `new_from_string'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/rack-2.0.5/lib/rack/builder.rb:40:in `parse_file'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/rack-2.0.5/lib/rack/server.rb:319:in `build_app_and_options_from_config'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/rack-2.0.5/lib/rack/server.rb:219:in `app'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/commands/server/server_command.rb:27:in `app'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/rack-2.0.5/lib/rack/server.rb:354:in `wrapped_app'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/rack-2.0.5/lib/rack/server.rb:283:in `start'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/commands/server/server_command.rb:53:in `start'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/commands/server/server_command.rb:147:in `block in perform'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/commands/server/server_command.rb:142:in `tap'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/commands/server/server_command.rb:142:in `perform'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/command/base.rb:65:in `perform'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/command.rb:46:in `invoke'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/railties-5.2.0/lib/rails/commands.rb:18:in `<main>'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/loaded_features_index.rb:65:in `register'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:283:in `block in require'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:249:in `load_dependency'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.2.0/lib/active_support/dependencies.rb:283:in `require'
	from /home/aguirrel/proyectos/dedipago-rails/bin/rails:9:in `<top (required)>'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/spring-2.0.2/lib/spring/client/rails.rb:28:in `load'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/spring-2.0.2/lib/spring/client/rails.rb:28:in `call'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/spring-2.0.2/lib/spring/client/command.rb:7:in `call'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/spring-2.0.2/lib/spring/client.rb:30:in `run'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/spring-2.0.2/bin/spring:49:in `<top (required)>'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/spring-2.0.2/lib/spring/binstub.rb:31:in `load'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/spring-2.0.2/lib/spring/binstub.rb:31:in `<top (required)>'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /home/aguirrel/.rbenv/versions/2.4.2/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /home/aguirrel/proyectos/dedipago-rails/bin/spring:15:in `<top (required)>'
	from bin/rails:3:in `load'
	from bin/rails:3:in `<main>'

```

### PR DESCRIPTION
**Pull request by Atul9**
closes #56

### CODE REVIEW

File: lib/invoicing/currency_value.rb [hunk 1]
```diff
@@ -119,7 +119,7 @@ def acts_as_currency_value(*args)
 
     included do
       # Register callback if this is the first time acts_as_currency_value has been called
-      before_save :write_back_currency_values, :if => "currency_value_class_info.previous_info.nil?"
+      before_save :write_back_currency_values
```
Comment (iffyuva) [inline lib/invoicing/currency_value.rb]: Why is this `if` condition removed @Atul9 ?

File: lib/invoicing/time_dependent.rb [hunk 2]
```diff
@@ -195,7 +195,7 @@ def acts_as_time_dependent(*args)
         # Create replaced_by association if it doesn't exist yet
         replaced_by_id = time_dependent_class_info.method(:replaced_by_id)
         unless respond_to? :replaced_by
-          belongs_to :replaced_by, :class_name => self, :foreign_key => replaced_by_id
+          belongs_to :replaced_by, :class_name => self.to_s, :foreign_key => replaced_by_id
```
Comment (iffyuva) [inline lib/invoicing/time_dependent.rb]: Lets use latest hash syntax, ie `class_name: `, and `foreign_key:`

**Comment by iffyuva** [comment]
@etcook we need to test this out, it will take sometime. Can you help us test this if possible?

**Comment by etcook** [comment]
@iffyuva sure - how can I help? Want to connect offline?

**Comment by etcook** [comment]
@iffyuva I've done a bit of integration and testing. This commit does indeed fix the Rails 5 issue referenced in #56

I'll open up a new issue with my thoughts for discussion.

File: lib/invoicing/currency_value.rb (hunk 1 above)
Comment (fonji) [reply lib/invoicing/currency_value.rb]: Use a proc or a bloc instead?
`if: { currency_value_class_info.previous_info.nil? }`

**Comment by PhilT** [comment]
Is this still an issue?

**Comment by weber-s** [comment]
Hello!

Is this branch going to be merge anytime soon? I have an application that rely on this gem, and I would like to update it to rails 5, so I'm looking forward the publication of this gem compatible for it :) 

Thanks for the job anyway!

File: lib/invoicing/currency_value.rb (hunk 1 above)
Comment (Atul9) [reply lib/invoicing/currency_value.rb]: done, I have updated the PR to use `if: currency_value_class_info.previous_info.nil?`

**Comment by Atul9** [comment]
Hi all, this PR is merged and I have published the gem to rubygems.org - https://rubygems.org/gems/invoicing/versions/1.1.0

**Comment by weber-s** [comment]
Thank you very much for the work!! (and the quick reply!!!)

**Comment by etcook** [comment]
@Atul9 This might be a localized issue for me - I haven't done any testing, but catching my fork up to this release broke my app, and I wanted to put a quick note here just in case it's not just localized to me.

`       NameError: undefined local variable or method `currency_value_class_info' for #<Class:0x00005610905e6d10>
       /tmp/build_321a9cace7dfa25e2030fa43705633ce/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.2.1/lib/active_record/dynamic_matchers.rb:22:in `method_missing'
       /tmp/build_321a9cace7dfa25e2030fa43705633ce/vendor/bundle/ruby/2.6.0/bundler/gems/invoicing-2aab0bdbba65/lib/invoicing/currency_value.rb:122:in `block in <module:CurrencyValue>'
       /tmp/build_321a9cace7dfa25e2030fa43705633ce/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/concern.rb:122:in `class_eval'
       /tmp/build_321a9cace7dfa25e2030fa43705633ce/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2.1/lib/active_support/concern.rb:122:in `append_features'
       /tmp/build_321a9cace7dfa25e2030fa43705633ce/vendor/bundle/ruby/2.6.0/bundler/gems/invoicing-2aab0bdbba65/lib/invoicing/class_info.rb:75:in `include'
       /tmp/build_321a9cace7dfa25e2030fa43705633ce/vendor/bundle/ruby/2.6.0/bundler/gems/invoicing-2aab0bdbba65/lib/invoicing/class_info.rb:75:in `acts_as'
       /tmp/build_321a9cace7dfa25e2030fa43705633ce/vendor/bundle/ruby/2.6.0/bundler/gems/invoicing-2aab0bdbba65/lib/invoicing/currency_value.rb:116:in `acts_as_currency_value'
       /tmp/build_321a9cace7dfa25e2030fa43705633ce/vendor/bundle/ruby/2.6.0/bundler/gems/invoicing-2aab0bdbba65/lib/invoicing/ledger_item.rb:315:in `acts_as_ledger_item'
       /tmp/build_321a9cace7dfa25e2030fa43705633ce/app/models/invoicing_ledger_item.rb:2:in `<class:InvoicingLedgerItem>'
       /tmp/build_321a9cace7dfa25e2030fa43705633ce/app/models/invoicing_ledger_item.rb:1:in `<main>'`

**Pull request merged**